modules/remoteapi/sc.php

```diff
@@ -12,9 +12,9 @@
         $billing->setcredit($userLogin, $newCredit);
         $billing->setcreditexpire($userLogin, $creditExpire);
         //charging some money
-        zb_CashAdd($userLogin, '-' . $creditFee, 'add', $creditCashType, 'SCFEE');
+        zb_CashAdd($userLogin, '-0' . $creditFee, 'add', $creditCashType, 'SCFEE');
```

Comment:
![photo_2023-01-27_11-02-08](https://user-images.githubusercontent.com/1496954/215047957-7a570bd0-125b-42db-8c24-685515ae3562.jpg)

Reply:
I'm sorry. thank you for your time.